addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBridgeHandler.java

```diff
@@ -250,13 +254,13 @@ public void responseReceived(RDVersionResponse response) {
                     });
 
         } catch (IOException e) {
-            logger.debug("error during bridge init occured", e);
+            logger.warn("error during bridge init occured", e);
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Port could not be found");
         } catch (PortInUseException e) {
-            logger.debug("error during bridge init occured", e);
+            logger.warn("error during bridge init occured", e);
```

Comment:
Please remove logging in front of thing updates. The framework already logs this and it is not of log level "warning" actually.

Reply:
Hmm I have mixed feelings because the logger carries additional information about the exception like the class name, exception message and stack trace, which the thing status does not contain. And because we catch the exception I don't think that it is logged by the framework, is it?

Nevertheless, I have removed the logging.

Reply:
If the stack trace doesn't add any additional information the logging doesn't indeed add anything as the status update is also logged. If the stack trace contains information that might be relevant to fix a problem that might be here due to the exception thrown than it can be useful to log the exception.